stdlib/array.gr

```diff
@@ -703,6 +777,9 @@ provide let counti = (fn, array) => {
  * @param fn: The function to call on each element, where the returned value indicates if the element satisfies the condition
  * @param array: The array to iterate
  * @returns The new array containing elements where `fn` returned `true`
+ * 
+ * @example
+ * Array.filter(a => a < 8, [> 5, 6, 7, 8, 9]) // [> 5, 6, 7]
```

Comment:
Same As Above:
```suggestion
 * Array.filter(a => a < 8, [> 5, 6, 7, 8, 9]) == [> 5, 6, 7]
```

Reply:
Same comment as before. This is not returning void so simply in line the result.....